test-packages/test-utils/api.d.ts

```diff
@@ -68,19 +69,23 @@ export function buildUI5Model(
   opts: Partial<UI5SemanticModel>
 ): UI5SemanticModel;
 
-export type TestModelVersion = "1.60.14" | "1.74.0" | "1.75.0";
+export type TestModelVersion = "1.60.14" | "1.74.0" | "1.75.0" | "1.71.14";
 
 /**
  * Return a UI5SemanticModel for the specified version.
+ *
+ * If downloadLibraries is true (default), increase the timeout of the test/hook to GEN_MODEL_TIMEOUT.
+ *
  * @param version
+ * @param downloadLibraries By default, download the library files before creating the model.
+ *        If you call downloadLibraries explicitly before this function, send false (it will improve the performance).
  * @param disableCache By default caching is used. This improves performance and is safe (since the model is immutable).
  */
 export function generateModel(
   version: TestModelVersion,
+  downloadLibraries?: boolean,
```

Comment:
**Important:**

Multiple optional flags (of same type) in public (even if test-packages) `API --> Config Object Pattern`

Reply:
Actually I think I'll remove the third flag since it's not in use (I thought I'll use it... but then I didn't so it's unnecessary)

Reply:
I'll change it to config object

Reply:
It is just a test-package, but config-objects are a best practice for public APIs 😄